build.go

```diff
@@ -233,6 +235,35 @@ func Build(f BuildFunc, options ...Option) {
 				return
 			}
 		}
+
+		if (apiVersion.GreaterThan(apiV05) || apiVersion.Equal(apiV05)) && len(layer.ExecD) > 0 {
+			execdDir := filepath.Join(layer.Path, "exec.d")
+			err = os.MkdirAll(execdDir, os.ModePerm)
+			if err != nil {
+				config.exitHandler.Error(err)
+				return
+			}
+
+			lexicalWidth := 1 + int(math.Log10(float64(len(layer.ExecD))))
+
+			for i, exe := range layer.ExecD {
+				if exists, err := fs.Exists(exe); !exists {
+					config.exitHandler.Error(fmt.Errorf("file %s does not exist. Be sure to include it in the buildpack.toml", exe))
+					return
+				} else if err != nil {
+					config.exitHandler.Error(err)
+					return
+				}
```

Comment:
I wanted to explicitly call out adding the file to `buildpack.toml` since that's an additional manual step on top of creating `cmd/new-command/main.go`. If it doesn't add value I'm happy to remove the additional branch.

Reply:
If that's what you want, you can check the error coming back from `fs.Copy` to see if it is `os.ErrNotExist`, something like the following:

```go
err = fs.Copy(exe, destination)
if err != nil {
  if errors.Is(os.ErrNotExist) {
    err = fmt.Errorf("file %s does not exist. Be sure to include it in the buildpack.toml", exe)
  }

  config.exitHandler.Error(err)
  return
}
```